after/ftplugin/go.vim

```diff
@@ -53,6 +53,9 @@ command! -nargs=* -range GoClearTagOptions lua require('go.struct_tag').clear_op
 " quicktype
 command! -nargs=* -complete=file GoQuickType lua require('go.quick_type').quick_type(<count>, <f-args>)
 
+" iferr
+command! -nargs=* -complete=file GoIfErr lua require('go.iferr').add_iferr()
```

Comment:
Does `GoIfErr` need any parameters? If it is not, this should be `-nargs=0`

Reply:
Actually no parameters needed, just removed it completely as the defaults are fine.